CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - SN's version and capacity is announced via the attributes automatically but can be overwritten explicitly (#2455, #602)
 - `peapod` command for `neofs-lens` (#2507)
 -  New CLI exit code for awaiting timeout (#2380)
+- To `neofs-cli` commands: `bearer create`, `object put` added `--lifetime` flag (#1574) 
+- To `neofs-cli` commands `session create`, `storagegroup put` added `--expired-at` flag (#1574)
```

Comment:
This is an `added` section, so it can be rephrased like
 * `--lifetime` flag to `bearer create` and `object put` CLI commands
 * `--expire-at` flag to `session create` and `storagegroup put` CLI commands